package.json

```diff
@@ -38,7 +38,8 @@
     "ci:test": "yarn build && yarn ci:lint && jest --runInBand && yarn clean",
     "ci:publish": "yarn build && yarn semantic-release && yarn clean",
     "semantic-release": "semantic-release",
-    "clean": "rm -rf ./build && rm ./boilerplate/.gitignore.template"
+    "clean": "rm -rf ./build && rm ./boilerplate/.gitignore.template",
+    "postinstall": "solidarity"
```

Comment:
This unfortunately seems to also run with `npm install -g ignite-cli`, which breaks the build because Solidarity is a devDependency. Bummer that there's not a way to run a `postinstall` without also capturing the prod installs too.